themes/materialize_contenta/css/materialize-drupal.css

```diff
@@ -8795,3 +8795,5 @@ button.picker__today:focus, button.picker__clear:focus, button.picker__close:foc
     margin-bottom: 5em;
   }
 }
+
+/*# sourceMappingURL=materialize-drupal.css.map */
```

Comment:
@dawehner it's easier to do a separate commit when using the GH merge button. :-P